deploy/examples/create-external-cluster-resources.py

```diff
@@ -969,6 +970,16 @@ def get_rbd_node_caps_and_entity(self):
 
         return caps, entity
 
+    def get_healthchecker_caps_and_entity(self):
+        entity = "client.healthchecker"
+        caps = {
+            "mgr": "allow command config",
+            "mon": "allow r, allow command quorum_status, allow command version",
+            "osd": "profile rbd-read-only, allow rwx pool=default.rgw.meta, allow r pool=.rgw.root, allow rw pool=default.rgw.control, allow rx pool=default.rgw.log,allow x pool=default.rgw.buckets.index",
```

Comment:
Rgw pool prefix is not always same it can be set by the flag,
                    `self._arg_parser.rgw_pool_prefix`

Reply:
can you suggest flag you mean to update?